codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:codesign/codesign.dart';
+import 'package:file/file.dart';
+import 'package:process/process.dart';
+
+/// Interface for classes that interact with files nested inside of [RemoteZip]s.
+abstract class FileVisitor {
+  const FileVisitor();
+
+  Future<void> visitEmbeddedZip(EmbeddedZip file, String parent, String entitlementParentPath);
```

Comment:
Docs for the interface methods?

Reply:
Sure, I removed these functions and saved them for later PRs, based on Casey's comment.